book/continuous_integration.md

```diff
@@ -0,0 +1,192 @@
+# Automating Code Quality Assurance
+
+## Introduction
+
+There are various tasks which can be automated to increase the quality of code and make development easier and less tedious. Automating the running of unit tests is especially important to ensuring trust in your pipeline or package, by ensuring that all unit tests pass before every merge.
+
+(continuous-integration)=
+## Continous Integration
```

Comment:
We should try to avoid defining ci/cd as it can be quite difficult to define.

Reply:
We've referred to CI in a number of places. It's a commonly used term so I'm not sure that removing it is the right approach.

I've update this section a little though to broaden the scope.